### PR TITLE
Adding title attributes

### LINKED ISSUE #89: Interactive elements have no titles or tooltips
It seems like the `aria-label` attribute is being used almost exclusively for adding labels to elements. No browsers that I'm aware of will use `aria-label` in a non-assistive context, so right now none of the buttons or other interactive elements will ever display a tooltip for most default browser settings. I think interactive elements should also get a `title` attribute so that users with non-assistive browsers can see some advisory information related to the elements.

### PR DESCRIPTION
**Pull request by a-maci29**
Closes #89 

- Includes `title` attributes for browser-supported assisted technology.

## To Review

- [ ] Review and ensure the `title` attribute is added correctly on all elements with an `aria-label` attribute.

### CODE REVIEW

File: components/Listen/EpisodeList/EpisodeList.tsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ const EpisodeList: React.FC<IEpisodeListProps> = ({
             index={index}
             onEpisodeClick={onEpisodeClick}
             key={track.guid}
+            title="test"
```
Comment (rpeterman-gp) [inline components/Listen/EpisodeList/EpisodeList.tsx:57]: Looks like some debugging code was left behind. This needs to be removed. It is causing build errors.

File: components/Listen/EpisodeList/EpisodeCard/EpisodeCard.tsx [hunk 2]
```diff
@@ -122,27 +121,24 @@ const EpisodeCard = ({ index, episode, onEpisodeClick }: IEpisodeCardProps) => {
...
               )}
             </span>
 
             {explicit && (
-              <span className={styles.explicit}>
-                <ExplicitIcon
-                  className={styles.explicit}
-                  aria-label="Explicit"
-                />
+              <span className={styles.explicit} aria-label="Explicit">
```
Comment (rpeterman-gp) [inline components/Listen/EpisodeList/EpisodeCard/EpisodeCard.tsx:140]: Lets move this `aria-label` back to the `<ExplicitIcon>` element.

File: components/Modal/Modal.tsx [hunk 3]
```diff
@@ -52,12 +52,13 @@ const Modal: React.FC<IModalProps> = ({
       <ThemeVars theme="Modal" cssProps={styles} />
       <div className={styles.root}>
         <IconButton
+          title={!isOpen ? 'Open Share Menu' : 'Close Share Menu'}
```
Comment (rpeterman-gp) [inline components/Modal/Modal.tsx:55]: Modal component can be used for more than just the Share menu. This title can just be "Close".

File: components/Player/PlayerThumbnail/PlayerThumbnail.tsx [hunk 4]
```diff
@@ -48,6 +48,7 @@ const PlayerThumbnail: React.FC<IPlayerThumbnailProps> = ({
   return (
     srcUrl && (
       <div
+        title={`Thumbnail for "${title}".`}
```
Comment (rpeterman-gp) [inline components/Player/PlayerThumbnail/PlayerThumbnail.tsx:51]: We don't need a title here. The alt text on the image is all we need for the thumbnail.

File: components/Player/SupportMenu/SupportMenu.tsx [hunk 5]
```diff
@@ -48,19 +48,20 @@ const SupportMenu: React.FC<ISupportMenuProps> = ({
         className={clsx(className)}
         href={followUrlsEntries[0][1]}
       >
-        <IconComponent aria-label={label} />
+        <IconComponent title={label} />
```
Comment (rpeterman-gp) [inline components/Player/SupportMenu/SupportMenu.tsx:51]: Move this title to the IconButton element.

File: components/Player/WebMonetized/WebMonetized.tsx [hunk 6]
```diff
@@ -204,7 +204,7 @@ const MebMonetized: React.FC<IMebMonetizedProps> = ({
         className={buttonClasses}
         onClick={handleClick}
       >
-        <MonetizationIcon aria-label="Web Monetization" />
+        <MonetizationIcon title="Web Monetization" />
```
Comment (rpeterman-gp) [inline components/Player/WebMonetized/WebMonetized.tsx:207]: Move this title to the IconButton element.

File: components/Player/WebMonetized/WebMonetized.tsx [hunk 7]
```diff
@@ -222,7 +222,7 @@ const MebMonetized: React.FC<IMebMonetizedProps> = ({
             <div className={styles.providers}>
               <h4>Web Monetization Providers</h4>
               <a href="https://coil.com" target="_blank" rel="noreferrer">
-                <CoilLogo className={styles.provider} aria-label="Coil" />
+                <CoilLogo className={styles.provider} title="Coil" />
```
Comment (rpeterman-gp) [inline components/Player/WebMonetized/WebMonetized.tsx:225]: This logo svg doesn't need a tooltip. This `title` should be a `aria-label`.

File: pages/embed.tsx [hunk 8]
```diff
@@ -241,7 +241,7 @@ const EmbedPage = ({ config, data, error }: IEmbedPageProps) => {
   }, [handleResize]);
 
   if (error) {
-    return <Error statusCode={error.statusCode} title={error.message} />;
```
Comment (rpeterman-gp) [inline pages/embed.tsx:244]: This title needs to stay. It is not the same as an HTML title attribute. It is the property that this component uses to provide the error message.

**Review by rpeterman-gp — CHANGES_REQUESTED** [review]
See comments for changes. Ping me with any questions.

File: components/Listen/EpisodeList/EpisodeCard/EpisodeCard.tsx (hunk 2 above)
Comment (a-maci29) [reply components/Listen/EpisodeList/EpisodeCard/EpisodeCard.tsx:140]: Moved it back to the `ExplicitIcon` element on line 141; is there a separate `ExplicitIcon` component?

**Review by rpeterman-gp — APPROVED** [review]

**Pull request merged**